src/lib/cookies.ts

```diff
@@ -0,0 +1,14 @@
+export function getCookie(name: string) {
```

Comment:
As we adding common cookies utils let's escape characters with `encodeURIComponent`, otherwise in future we may set cookie value which can broke this code

Reply:
oh, we don't use cookie eventually, i've just deleted this file